pkg/agent/testdata/AKSWindows2004+K8S119+hyperv/CustomData

```diff
@@ -0,0 +1,491 @@
+
+$global:UseManagedIdentityExtension = "false"
+$global:UseInstanceMetadata = "true"
+
+$global:LoadBalancerSku = "Standard"
+$global:ExcludeMasterFromStandardLB = "true"
+
+
+# Windows defaults, not changed by aks-engine
+$global:CacheDir = "c:\akse-cache"
+$global:KubeDir = "c:\k"
+$global:HNSModule = [Io.path]::Combine("$global:KubeDir", "hns.psm1")
+
+$global:KubeDnsSearchPath = "svc.cluster.local"
+
+$global:CNIPath = [Io.path]::Combine("$global:KubeDir", "cni")
+$global:NetworkMode = "L2Bridge"
+$global:CNIConfig = [Io.path]::Combine($global:CNIPath, "config", "`$global:NetworkMode.conf")
+$global:CNIConfigPath = [Io.path]::Combine("$global:CNIPath", "config")
+
+
+$global:AzureCNIDir = [Io.path]::Combine("$global:KubeDir", "azurecni")
+$global:AzureCNIBinDir = [Io.path]::Combine("$global:AzureCNIDir", "bin")
+$global:AzureCNIConfDir = [Io.path]::Combine("$global:AzureCNIDir", "netconf")
+
+# Azure cni configuration
+# $global:NetworkPolicy = "" # BUG: unused
+$global:NetworkPlugin = "azure"
+$global:VNetCNIPluginsURL = "https://acs-mirror.azureedge.net/azure-cni/v1.1.3/binaries/azure-vnet-cni-singletenancy-windows-amd64-v1.1.3.zip"
+$global:IsDualStackEnabled = $false
+
+# Telemetry settings
+$global:EnableTelemetry = [System.Convert]::ToBoolean("false");
+$global:TelemetryKey = "";
+
+# CSI Proxy settings
+$global:EnableCsiProxy = [System.Convert]::ToBoolean("false");
+$global:CsiProxyUrl = "";
+
+# Hosts Config Agent settings
+$global:EnableHostsConfigAgent = [System.Convert]::ToBoolean("false");
+
+$global:ProvisioningScriptsPackageUrl = "https://acs-mirror.azureedge.net/aks-engine/windows/provisioning/signedscripts-v0.0.4.zip";
```

Comment:
where/who create this file? why is it under aks-engine?

Reply:
upstream signed and published the script, it's used in production as you can see in https://github.com/Azure/AgentBaker/blob/master/vhdbuilder/packer/configure-windows-vhd.ps1#L105